pof/src/parse.rs

```diff
@@ -1642,7 +1679,14 @@ pub fn parse_gltf(path: std::path::PathBuf) -> Model {
         })
         .collect();
 
-    GltfContext { buffers }.parse_top_level_nodes(&mut model, gltf.default_scene().expect("Default scene in gltf file not set!").nodes());
+    let scene = gltf
+        .default_scene()
+        .unwrap_or(gltf.scenes().next().expect("no scene found in gltf file!"));
```

Comment:
```suggestion
        .unwrap_or_else(|| gltf.scenes().next().expect("no scene found in gltf file!"));
```

Reply:
Why?

Reply:
Well the function calls there are unnecessary, but in particular it seems bad to run something that can panic. I believe there is a clippy warning for this: always use delayed evaluation in this situation unless the default is just a plain value (e.g. a literal or a struct constructor expression of such)